tests/installation/accept_license.pm

```diff
@@ -35,16 +35,7 @@ sub run {
     push @tags, 'inst-welcome-no-product-list' if is_sle('15+') && get_var('UPGRADE');
 
     assert_screen \@tags;
-    if (match_has_tag('network-settings-button')) {
```

Comment:
The ones used in removed piece of code, namely with network-settings-button tag. These are the only usages, so also, please, clean-up  tags array on the line 27

Reply:
tag comment below ...